site/content/tutorial/19-next-steps/01-congratulations/text.md

```diff
@@ -6,6 +6,6 @@ You've now finished the Svelte tutorial and are ready to start building apps. Yo
 
 To get set up in your local development environment, check out [the quickstart guide](blog/the-easiest-way-to-get-started).
 
-If you're looking for a more expansive framework that includes routing, server-side rendering and everything else, take a look at [Sapper](https://sapper.svelte.dev).
+If you're looking for a more expansive framework that includes routing, server-side rendering and everything else, take a look at [Svelte Kit](https://kit.svelte.dev).
```

Comment:
If you're looking for a more expansive framework that includes routing, server-side rendering and everything else, take a look at [Sapper](https://sapper.svelte.dev) and [Svelte Kit](https://kit.svelte.dev)

Reply:
Good point, [done](https://github.com/sveltejs/svelte/pull/6284/commits/ded5af31ca111db3b6537dfc3cf873bc945455ea).

Reply:
Sapper is basically EOL at this point. I think the original edit was preferable

Reply:
Reverted back to just reference Svelte Kit